releases/cockroachcloud-12092020.md

```diff
@@ -0,0 +1,28 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+---
+
+## December 9, 2020
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### General changes
+
+New CockroachCloud clusters will now run CockroachDB [v20.2.2](v20.2.2.html).
```

Comment:
@emily-horing is this the correct version? Also, can you review the language explaining the machine sizing / pricing?

Reply:
Yep all new clusters are running CRDB 20.2.2 as of 12/2 per [this thread](https://cockroachlabs.slack.com/archives/CJX8V9SJ2/p1607545738181300)

For machine sizes you could say "CockroachCloud is now offering larger machines to be configured directly in the Console". 
The pricing was changed for more than just the large sizes in the latest update so maybe also adding something like "The pricing has also been updated for newly created clusters. Existing clusters are not impacted by pricing changes." Something like that?

Reply:
Updated, thanks!